compiler/rustc_passes/src/reachable.rs

```diff
@@ -1,9 +1,15 @@
-// Finds items that are externally reachable, to determine which items
-// need to have their metadata (and possibly their AST) serialized.
-// All items that can be referred to through an exported name are
-// reachable, and when a reachable thing is inline or generic, it
-// makes all other generics or inline functions that it references
-// reachable as well.
```

Comment:
The old text said "it makes all all other generics or inline functions that it references reachable as well" -- but I don't think that matches the code? When a reachable thing is inline or generic, then it makes *all* other functions it references reachable as well. Not just those other functions it references that are themselves inline or generic. Or did I misunderstand?